DataConnectors/SentinelOne/SentinelOneSentinelConnector/__init__.py

```diff
@@ -39,14 +43,31 @@ def __init__(self):
         self.from_date, self.to_date = self.generate_date()
         self.results_array = []
         self.threat_id_arr = []
+        self.computerName_arr = []

```

Comment:
Use sanke_case for variable and function names.

Reply:
yes will change the naming.